journal/src/View/Converter/Block/AsideConverter.php

```diff
@@ -27,18 +30,24 @@ public function convert($object, array $context = []) : string
         if ($id = $object->getAttribute('id')) {
             $attributes .= " id=\"{$id->toText()}\"";
         }
-        if ($object->getAttribute('lang') ?? $context['lang'] !== $context['lang']) {
-            $attributes .= " lang=\"{$object->getAttribute('lang')->toText()}\"";
+        if ($object->getAttribute('lang') && $object->getAttribute('lang')->toText() !== $context['lang']) {
             $context['lang'] = $object->getAttribute('lang')->toText();
+            $dir = $this->translator->trans('direction', [], null, $context['lang']);
+
+            $attributes .= " lang=\"{$context['lang']}\"";
+            $attributes .= " dir=\"{$dir}\"";
```

Comment:
Ideally wouldn't repeat if the context is already the same direction, but I'm ignoring it for now.